docs/cookbook/deploying-to-heroku.md

```diff
@@ -0,0 +1,222 @@
+# Deploying Apostrophe in the Cloud with Heroku
+
+There are many cloud hosting services, but they all present the same challenges. For example, separate servers often don't share a single filesystem. The database usually needs its own scalable cloud hosting. And performing tasks like minifying assets is often best done in your development environment, minimizing what has to be done in production.
+
+
+## Deploying Apostrophe to Heroku
+
+[Heroku](http://heroku.com) is a great starting point for cloud hosting because it is simple to set up, but all of the cloud's challenges come into play. What we learn by deploying to Heroku can be applied equally to Amazon EC2, Microsoft Azure, and other cloud hosting services.
+
+To reduce costs for this how-to, we'll be using free services from Amazon Web Services and MongoDB Atlas, a MongoDB cloud hosting service from the creators of MongoDB. But remember that you can choose paid plans with much higher capacity and performance. Everything in this tutorial is designed to scale smoothly to those paid offerings. Heroku's offerings are in flux. They are eliminating their free tier, but they are offering low-price 'Eco' and 'Basic' plans that have the same provisioning as their free tier that was used to write this tutorial.
```

Comment:
Interesting, didn't know that.

as their free => as the free (better grammar? I think?)

Reply:
Changed.